common/lib/xmodule/xmodule/modulestore/django.py

```diff
@@ -74,6 +89,8 @@ def create_modulestore_instance(engine, content_store, doc_store_config, options
     if issubclass(class_, BranchSettingMixin):
         _options['branch_setting_func'] = _get_modulestore_branch_setting
 
+    xb_user_service = DjangoXBlockUserService(get_current_user()) if HAS_USER_SERVICE and not user_service else None
```

Comment:
Nit: I probably would have written this on two lines, but that's more a personal preference than a strong recommendation.